extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/AbstractWidgetRenderer.java

```diff
@@ -202,14 +208,13 @@ protected String processColor(Widget w, String snippet) {
     }
 
     protected String getCategory(Widget w) {
-        String category = escapeURL(itemUIRegistry.getCategory(w));
-        return category;
+        return itemUIRegistry.getCategory(w);
     }
 
     protected String getState(Widget w) {
         State state = itemUIRegistry.getState(w);
         if (state != null) {
-            return escapeURL(state.toString());
```

Comment:
There are no other callers but I think semantically it makes sense to return the state as is, because there's nothing about this method name suggesting it returns a state escaped to use within an URL.